docs/configuration/index.md

```diff
@@ -1801,7 +1801,7 @@ These Broker configurations can be defined in the `broker/runtime.properties` fi
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.query.scheduler.numThreads`|Maximum number of HTTP threads to dedicate to query processing. To save HTTP thread capacity, this should be lower than `druid.server.http.numThreads`. It is worth noting that when `druid.server.http.enableRequestLimit` is set, query requests over this limit will be denied instead of waiting in the Jetty HTTP request queue.|Unbounded|
+|`druid.query.scheduler.numThreads`|Maximum number of HTTP threads to dedicate to query processing. To save HTTP thread capacity, this should be lower than `druid.server.http.numThreads`. When this parameter is set, query requests over the limit are denied with HTTP 429 instead of waiting in the Jetty request queue.|Unbounded|
```

Comment:
> To save HTTP thread capacity

Should we also point out why we want to save the HTTP thread capacity?

Reply:
Updated wording a bit, LMK what you think.